core/src/apps/ethereum/helpers.py

```diff
@@ -126,6 +129,74 @@ def decode_typed_data(data: bytes, type_name: str) -> str:
     raise ValueError  # Unsupported data type for direct field decoding
 
 
+def get_fee_items_regular(
+    gas_price: int, gas_limit: int, network: EthereumNetworkInfo
+) -> Iterable[tuple[str, str]]:
+    # regular
+    gas_limit_str = TR.ethereum__units_template.format(gas_limit)
+    gas_price_str = format_ethereum_amount(
+        gas_price, None, network, force_unit_gwei=True
+    )
+
+    items = (
```

Comment:
I guess no need to initialize the variable, when we can just return its content right away

Reply:
https://github.com/trezor/trezor-firmware/pull/3534/commits/b3eb30b259e2034a10e613d2f20c7761538a91f7